torch/onnx/ops/_impl.py

```diff
@@ -0,0 +1,113 @@
+import typing
+from typing import Any, Callable, Optional
+
+import torch
+
+
+_T = typing.TypeVar("_T", bound=Callable)
+
+_ONNX_DECOMP_TABLE: dict[Any, Callable] = {}
+
+
+def onnx_aten_decomp_table() -> dict[Any, Callable]:
+    """Return the ONNX to ATen decomp table."""
+    return _ONNX_DECOMP_TABLE
```

Comment:
Can you elaborate on the naming. I find this is confusing because it makes me think of decomposition table we have for ATen -> ONNX. But this one looks like we use torch to implement ONNX op.

BTW, @xadupre might be able to keep his torch implementations of the ONNX ops from phi4_mm here that we can use for future validation?

Reply:
It’s a decomp table that can be used with run_decompositions. I added tests and documentation to demonstrate